source/components/Profile/styles.js

```diff
@@ -28,7 +28,25 @@ export default makeStyles({
   menu: {
     zIndex: 1,
   },
+  drawer: {
+    zIndex: '500 !important',
```

Comment:
why the `!important` ?

Reply:
because the material drawer itself adds styles when rendering, so even if i use a class it is overwritten by the style, so i have to add the important